how-to/customize-workspace/public/manifest.fin.json

```diff
@@ -680,18 +680,15 @@
 					"description": "Search for emojis by name",
 					"enabled": false,
 					"autoStart": false,
-					"moduleUrl": "https://samples.openfin.co/workspace-starter/workspace/v9.0.0/customize-home-templates/js/integrations/emoji.bundle.js",
-					"data": {}
+					"url": "https://samples.openfin.co/workspace-starter/workspace/v9.0.0/customize-home-templates/js/integrations/emoji.bundle.js"

```

Comment:
does this module need to match the new api now?

Reply:
Yes the modules will need updating